front/app/containers/Admin/projects/routes.tsx

```diff
@@ -215,160 +229,187 @@ const createAdminProjectsRoutes = () => {
-              </PageLoading>
-            ),
-          },
-          {
-            path: projectsRoutes.projectPhaseAccessRights,
-            element: (
-              <PageLoading>
-                <AdminProjectPermissions />
-              </PageLoading>
-            ),
-          },
-          {
-            path: projectsRoutes.projectPhaseIdeas,
-            element: (
-              <PageLoading>
-                <AdminProjectIdeas />
-              </PageLoading>
-            ),
-          },
-          {
-            path: projectsRoutes.projectPhaseIdeaForm,
-            element: (
-              <PageLoading>
-                <AdminProjectIdeaForm />
-              </PageLoading>
-            ),
-          },
-          {
-            path: projectsRoutes.projectPhaseVolunteering,
-            element: (
-              <PageLoading>
-                <AdminProjectVolunteering />
-              </PageLoading>
-            ),
-          },
-          {
-            path: projectsRoutes.projectPhaseMap,
-            element: (
-              <PageLoading>
-                <AdminCustomMapConfigComponent />
-              </PageLoading>
-            ),
-          },
-          {
-            path: projectsRoutes.projectPhaseVolunteeringNewCause,
-            element: (
-              <PageLoading>
-                <AdminProjectVolunteeringNew />
-              </PageLoading>
-            ),
-          },
-          {
-            path: projectsRoutes.projectPhaseNativeSurvey,
-            element: <AdminProjectsSurvey />,
-          },
-          {
-            path: projectsRoutes.projectPhaseIdeaFormEdit,
-            element: <IdeaFormBuilder />,
-          },
-          {
-            path: projectsRoutes.projectPhaseNativeSurveyEdit,
-            element: <SurveyFormBuilder />,
-          },
-          {
-            path: projectsRoutes.projectPhaseVolunteeringCause,
-            element: (
-              <PageLoading>
-                <AdminProjectVolunteeringEdit />
-              </PageLoading>
-            ),
-          },
-          // {
-          //   path: 'allowed-input-topics',
-          //   element: <AdminAllowedTopicsComponent />,
-          // },
-          {
-            path: projectsRoutes.projectAnalysis,
+            path: projectsRoutes.projectTraffic,
             element: (
               <PageLoading>
-                <AdminProjectAnalysis />
+                <ProjectTraffic />
               </PageLoading>
             ),
           },
           {
-            path: projectsRoutes.projectPhaseOfflineInputs,
+            path: projectsRoutes.projectParticipation,
             element: (
               <PageLoading>
-                <OfflineInputImporter />
+                <ProjectParticipation />
               </PageLoading>
             ),
           },
           {
-            path: projectsRoutes.projectPhaseReport,
+            path: projectsRoutes.projectIdPhases,
```

Comment:
This looks like a big change but all I did was create a second level of nesting to be able to implement the new routes. 
![Uploading image.png…]()